lib/bugsnag/tasks/bugsnag.rake

```diff
@@ -6,7 +6,14 @@ namespace :bugsnag do
     begin
       raise RuntimeError.new("Bugsnag test exception")
     rescue => e
-      Bugsnag.notify(e, {:context => "rake#test_exception"})
+      Bugsnag.auto_notify(e, {
```

Comment:
This is fine to just be a normal notify, doesnt need to be an auto_notify. Its just a way of people testing their bugsnag installations